### ISSUE TITLE
upgrade to the latest object-keys

### ISSUE THREAD

**Issue by jonathanong** [Raynos/xtend#30]
https://github.com/ljharb/object-keys

right now, npm logs:

```bash
npm WARN deprecated object-keys@0.2.0: Please update to the latest object-keys
```

**Comment by TehShrike** [comment]
This fix was published in 2.2.0. If you upgrade to a newer version of xtend, you should be good.

https://github.com/Raynos/xtend/commit/5c61e0b9dfe6cc9e14bf95829312d3e47adca26a

**Comment by jonathanong** [comment]
oh hmm. i guess i opened this without checking the current version. it's a 3rd party module that's using an old version of this module. :(
